int64 index
float64 distance
float64 azimuth
float64 confidence
